source/conf.py

```diff
@@ -37,6 +37,31 @@
     ".md": "markdown",
 }
 
+# -- Options for linkcheck ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder
+
+linkcheck_timeout = 30
+linkcheck_workers = 5
+linkcheck_retries = 3
+linkcheck_ignore = [
+    r'http://localhost.*',
+    r'http://127\.0\.0\.1.*',
+    r'.*ekxide\.io.*',
+]
+
+linkcheck_rate_limit_timeout = 30.0
+linkcheck_anchors = True
+
+import os
+linkcheck_request_headers = {
+    "*": {
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:100.0) Gecko/20100101 Firefox/100.0",
+    },
+    "github.com": {
+        "Authorization": f"token {os.environ.get('GITHUB_TOKEN', '')}",
```

Comment:
No sure why we need the token here, that should normally work without authentication.